mlflow/models/model.py

```diff
@@ -613,6 +614,23 @@ def log(
                 run_id = mlflow.tracking.fluent._get_or_start_run().info.run_id
             mlflow_model = cls(artifact_path=artifact_path, run_id=run_id, metadata=metadata)
             flavor.save_model(path=local_path, mlflow_model=mlflow_model, **kwargs)
+
+            # Copy metadata files to the 'metadata' subdirectory
```

Comment:
Aren't the only metadata files `MLmodel` and `input_example.json`? Why not just copy those into the `metadata` directory / why define data paths in each flavor?

Reply:
Ah I see in the design doc there are some others: https://docs.google.com/document/d/1t6zx_S12N9iDM2cRpeLJrzMtJu-awVdZmpgMFzAPJ_4/edit#bookmark=id.6zf1wbhmhnai

Reply:
But wouldn't be easier to mark those as metadata than try to mark the model data files in each flavor? Seems like most flavors don't log additional metadata files (only a handful like `transformers` do)

Reply:
I'm of the opinion that explicitly defining these is safer for newer flavor creation so that a new implementation doesn't inadvertently create a security issue for metadata sharing in UC. While it does look like a lot of copying of functionality for most flavors, I think this might be safer to avoid the chance of forgetting to update a blacklist instead of this approach which is explicitly whitelisting within the scope of each flavor implementation. 
This was the primary reason for going this route.